x/inter-tx/types/query.go

```diff
@@ -0,0 +1,17 @@
+package types
+
+// NewQueryInterchainAccountRequest creates and returns a new QueryInterchainAccountFromAddressRequest
+func NewQueryInterchainAccountRequest(owner, connectionID, counterpartyConnectionID string) *QueryInterchainAccountFromAddressRequest {
+	return &QueryInterchainAccountFromAddressRequest{
+		Owner:                    owner,
+		ConnectionId:             connectionID,
+		CounterpartyConnectionId: counterpartyConnectionID,
```

Comment:
We can probs remove the counterparty connection here also (doesn't need to be in this PR) just food for thought.

Reply:
Agreed